storage/pruning/pruningStorer.go

```diff
@@ -1033,6 +1033,11 @@ func (ps *PruningStorer) RangeKeys(_ func(key []byte, val []byte) bool) {
 	debug.PrintStack()
 }
 
+// GetIdentifier returns the identifier for storer
+func (ps *PruningStorer) GetIdentifier() string {
+	return ps.identifier
```

Comment:
I think that if we somehow, change the paths and names for the accounts storers, the whole code will break. We need to find a proper way to use storers identifiers in this case. Perhaps using the trie containers implementation as seen in factory/state/stateComponents.go L151-L178?
This happens because we have 2 sets of constants declared: one in trie/trieFactory/trieFactoryArgs.go and the other one in dataRetriever/unitType.go

Reply:
from what i've tested, it still seems to work, will think more on the refactoring here

Reply:
Works for now because the strings from both constant sets are the same. Will need to refactor this somehow. You can refactor now or add a TODO + task.

Reply:
added todo and separate task for this